lib/node_modules/@stdlib/stats/base/svarianceyc/include/stdlib/stats/base/svarianceyc.h

```diff
@@ -31,7 +31,12 @@ extern "C" {
 /**
 * Computes the variance of a single-precision floating-point strided array using a one-pass algorithm proposed by Youngs and Cramer.
 */
-float stdlib_strided_svarianceyc( const int64_t N, const float correction, const float *X, const int64_t stride );
+float API_SUFFIX(stdlib_strided_svarianceyc)( const CBLAS_INT N, const float correction, const float *X, const CBLAS_INT strideX );
+
+/**
+* Computes the variance of a single-precision floating-point strided array using a one-pass algorithm proposed by Youngs and Cramer, using alternative indexing semantics.
```

Comment:
```suggestion
* Computes the variance of a single-precision floating-point strided array using a one-pass algorithm proposed by Youngs and Cramer and alternative indexing semantics.
```

this is the convention that we're following across the `stats/base` namespace for now